EngineeringModel.Tests/Services/ParameterActualFiniteStateListApplicationBatchServiceTestFixture.cs

```diff
@@ -0,0 +1,121 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ParameterActualFiniteStateListApplicationBatchServiceTestFixture.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2020 RHEA System S.A.
+//
+//    Author: Sam Gerené, Alex Vorobiev, Naron Phou, Alexander van Delft, Nathanael Smiechowski, Ahmed Ahmed
```

Comment:
@ahmedatrhea can you please tell us the preferred way?

Reply:
I would keep "Ahmed Abulwafa" to avoid confusion, but my last name is actually Ahmed, so if we "have to" keep it to that convention so I think "Ahmed Abulwafa Ahmed" is the way to go.